source/beamline/BioXAS/BioXASFrontEndShutters.cpp

```diff
@@ -46,7 +45,7 @@ void BioXASFrontEndShutters::setUpstreamPhotonShutter(AMReadOnlyPVControl *newCo
 		upstreamPhotonShutter_ = newControl;
 
 		if (upstreamPhotonShutter_)
-			addShutter(upstreamPhotonShutter_, 1, 4);
+			addShutter(upstreamPhotonShutter_, CLS_SHUTTER_OPEN, CLS_SHUTTER_CLOSED);
```

Comment:
@helfrij yes, that confused me before. the Open/Close value we used in the addShutter function should be the one we used for the control.

That is, if the control is a regular PV, the open/close should be 1/4 as the case here. If the control is a pseudomotor, the open/close should be 0/1 (or 1/0) (this depends on the open/close definition of that pseudomotor).